iptables/iptables.go

```diff
@@ -369,3 +376,19 @@ func ExistChain(chain string, table Table) bool {
 	}
 	return false
 }
+
+// GetVersion reads the iptables version numbers
+func GetVersion() (major, minor, micro int, err error) {
```

Comment:
this again goes to the same discussion on checking for versions vs checking for functionality (we had a few such scenarios for kernel versions and we typically prefer the functionality check).

@mrjana WDYT ?